asmcomp/arm64/emit.mlp

```diff
@@ -27,6 +27,9 @@ open Mach
 open Linear
 open Emitaux
 
+let is_macosx =
+  Config.system = "macosx"
```

Comment:
You could move this definition to `asmcomp/arm64/arch.ml`, so that you don't have to repeat it in selection.ml.

Also, the house style would omit `is_` and call this variable `macosx`.  For example in `asmcomp/amd64/arch.ml`, we have a `win64` Boolean, not `is_win64`.

Also, the new brand name is macOS, so `macOS` or `macos` would work too.

Reply:
I'm trying to keep it coherently with every other mention where it is `macosx`, if we want to rename it it should be in a future PR. But fully agreed on removing the `is_` and moving to `arch.ml`